torax/geometry.py

```diff
@@ -879,6 +880,213 @@ def from_fbt(
         hires_fac=hires_fac,
     )
 
+  @classmethod
+  def from_eqdsk(
+        cls,
+        geometry_dir: str | None = None,
+        geometry_file: str = 'eqdsk_cocos02.eqdsk',
+        hires_fac: int = 4,
+        Ip_from_parameters: bool = True,
+        n_rho: int = 25,
+        # NOTE: The other geometry factors are coming from eqdsk...
+    ):
+        # helper function to calculate area of a polygon given curve along x, z
+        def calculate_area(x, z):
+            # Gauss-shoelace formula (https://en.wikipedia.org/wiki/Shoelace_formula)
+            # could likely use the determinant approach
+            #  0.5 * np.abs(np.dot(x, np.roll(z, 1)) - np.dot(z, np.roll(x, 1)))
```

Comment:
Is this an alternative method, if we aren't using this I am not sure we need to add this comment